CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.21.2 - 2021-03-09
+### [Fixed](https://github.com/microsoft/vscode-azureappservice/issues/1571)
+- Deployments on WSL and Codespaces should no longer hang and fail by changing archiver
```

Comment:
The "by changing archiver" part threw me off a bit. What about "Changed archiver to fix deployment failures on WSL and Codespaces"?